Add (le|sa|sc)ss and styl file types to list of forced text extensions.

rowild opened the pull request:
<!-- Is it a Bug fix, feature, docs update, ... -->
**What kind of change does this PR introduce?**
Add (le|sa|sc)ss and style files types to the list of forced text extensions.


<!-- You can also link to an open issue here -->
**What is the current behavior?**
https://github.com/CompuIves/codesandbox-client/issues/348#issuecomment-416019773
When uploading(le|sa|sc)ss or styl (and probably other) files, and error message is thrown:

"This file is too big to edit
We will add support for this as soon as possible
Open file externally". 

It seems the zip implementation does not recognize them as text files. CompuIves recommends to add them to the list of forced text extensions.

Important: This happens only, when such files are *uploaded*! (creating such files within the editor works fine.)

<!-- if this is a feature change -->
**What is the new behavior?**
On upload, recognize more file types. (Again: on *upload*!)

<!-- Have you done all of these things?  -->
**Checklist**:
<!-- add "N/A" to the end of each line that's irrelevant to your changes -->
<!-- to check an item, place an "x" in the box like so: "- [x] Documentation" -->
- [ ] Documentation N/A
- [ ] Tests
- [ ] Ready to be merged <!-- In your opinion, is this ready to be merged as soon as it's reviewed? -->
- [ ] Added myself to contributors table <!-- this is optional, see the contributing guidelines for instructions -->

<!-- feel free to add additional comments -->
I do this upon being challenged by compulves (see link above) to contribute.

<!-- Thank you for contributing! -->

CompuIves commented:
This looks really good, thanks a lot! Could you also add the `css` extension? Then we're ready to go!

rowild commented:
But regular CSS (as well as js) work fine (unless they are large minified once). I test with the repo in the git post (here is the link again: https://codesandbox.io/s/lp2o446nrz)
Do you nevertheless want me to add them? Maybe checks against .coffee, .typescript, haml, pug (if they have any) should also be done.

CompuIves commented:
Oh no that's great then! I think we could add a check for `.haml` and `.pug` and then merge this in. You can mark yourself as contributor (also on CodeSandbox) by running `yarn add-contributor`.

rowild commented:
Added .haml, .pug and .coffee. .typescript is already checked, as I realized.
The CI still returns errors...
Do you prefer if I added myself as a contributor? Then I should fork the repo to the desktop, right? Because right now it is only a fork on github...

CompuIves commented on packages/app/src/app/store/modules/files/actions.js:
```diff
@@ -68,9 +68,14 @@ export async function uploadFiles({ api, props, path }) {
               /\.html$/.test(filePath) ||
               /\.vue$/.test(filePath) ||
+              /\.styl$/.test(filePath) ||
+              /\.(le|sc|sa)ss$/.test(filePath) ||
```
Yep! This looks good 👍

rowild commented:
Awesome! When will it be available in the online version? (Because at the moment it does not work...)
